infrastructure/smart-contracts/scripts/tiny-keys/tinyKeysAirdropDeployer.mjs

```diff
@@ -10,18 +10,34 @@ async function main() {
 
     // get the deployer
     const [deployer] = (await ethers.getSigners());
-    const deployerAddress = await deployer.getAddress();
+    console.log("Deploying contracts with the account:", deployer.address);
```

Comment:
How come we do `await signer.getAddress()` when we could have been doing `signer.address`. If this works just fine we should be conscious about any new code that keeps doing await getAddress right ?

Reply:
Confirmed this prints properly on deployment.